### PR TITLE
fix: [CDS-50426]: Show Visual View for new trigger creation

### PR DESCRIPTION
**Pull request by pankajsingh-harness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by pankajsingh-harness** [comment]
retrigger jest

File: src/modules/10-common/components/Wizard/Wizard.tsx [hunk 1]
```diff
@@ -121,6 +121,13 @@ const Wizard: React.FC<WizardProps> = ({
   const [validateOnChange, setValidateOnChange] = React.useState<boolean>(formikInitialProps.validateOnChange || false)
   const [selectedTabIndex, setSelectedTabIndex] = React.useState<number>(initialIndex)
   const [selectedView, setSelectedView] = useTriggerView()
+  useEffect(() => {
+    // If the user preference stored in local storage is YAML, set the Visual View for the new trigger.
+    if (!isEdit && selectedView === SelectedView.YAML) {
+      setSelectedView(SelectedView.VISUAL)
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [isEdit])
```
Comment (rameshx) [inline src/modules/10-common/components/Wizard/Wizard.tsx:130]: Is it possible to move this effect inside useTriggerView to avoid repetition?

Comment (pankajsingh-harness) [reply src/modules/10-common/components/Wizard/Wizard.tsx:130]: Done.

**Review by rameshx — APPROVED** [review]

**Comment by pankajsingh-harness** [comment]
Only Coverage is failing 

Screenshot 2023-03-29 at 3 08 36 PM

**Pull request merged**